roles/download/defaults/main.yml

```diff
@@ -29,10 +29,10 @@ kubeadm_version: "{{ kube_version }}"
 etcd_version: v3.2.4
 # TODO(mattymo): Move calico versions to roles/network_plugins/calico/defaults
 # after migration to container download
-calico_version: "v2.5.0"
-calico_ctl_version: "v1.5.0"
-calico_cni_version: "v1.10.0"
-calico_policy_version: "v0.7.0"
+calico_version: "v2.6.2"
```

Comment:
There's 0 difference between 2.6.1 and 2.6.2 according to the release notes. Not asking you to change this, however.

Reply:
Yes, I tested a deployment with the updated values.

Reply:
There are also updates available for other calico components.
+calico_ctl_version: "v1.6.1"
+calico_cni_version: "v1.11.0"
+calico_policy_version: "v1.0.0"

Reply:
@skyscooby those are already updated inside this commit